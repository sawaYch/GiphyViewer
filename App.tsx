import React from 'react';
import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import { SafeAreaProvider } from 'react-native-safe-area-context';
import GiphyViewer from './src/components/GiphyViewer';
import AppThemeProvider from './src/components/AppThemeProvider';

const queryClient = new QueryClient();

const App = () => {
  return (
    <QueryClientProvider client={queryClient}>
      <SafeAreaProvider>
        <AppThemeProvider>
          <GiphyViewer />
        </AppThemeProvider>
      </SafeAreaProvider>
    </QueryClientProvider>
  );
};

export default App;
