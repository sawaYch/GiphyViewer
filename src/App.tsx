import React from 'react';
import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import { SafeAreaProvider } from 'react-native-safe-area-context';
import GiphyViewer from './components/GiphyViewer';
import { AppThemeProvider } from './components';

const queryClient = new QueryClient();

const App = () => {
  return (
    <QueryClientProvider client={queryClient}>
      <SafeAreaProvider>
        <AppThemeProvider>
          <GiphyViewer />
        </AppThemeProvider>
      </SafeAreaProvider>
    </QueryClientProvider>
  );
};

export default App;
