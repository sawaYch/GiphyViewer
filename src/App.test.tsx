import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('should render the app name on Header', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
