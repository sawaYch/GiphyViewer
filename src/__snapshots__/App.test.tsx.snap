// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the app name on Header 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
